BuildResidentialHPXML/measure.rb

```diff
@@ -1741,12 +1736,12 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
 
     arg = OpenStudio::Measure::OSArgument::makeStringArgument('hvac_control_heating_season_period', false)
     arg.setDisplayName('HVAC Control: Heating Season Period')
-    arg.setDescription("Enter a date like 'Nov 1 - Jun 30'. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#hpxml-hvac-control'>HPXML HVAC Control</a>) is used. Can also provide '#{HPXML::BuildingAmerica}' to use automatic seasons from the Building America House Simulation Protocols.")
+    arg.setDescription("Enter a date like 'Nov 1 - Jun 30'. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#hpxml-hvac-control'>HPXML HVAC Control</a>) is used. Can also provide '#{Constants::BuildingAmerica}' to use automatic seasons from the Building America House Simulation Protocols.")
```

Comment:
@joseph-robertson Does ResStock use this BA season option? I assume we had a good reason to add it to OS-HPXML directly, but ideally ResStock would be able to apply the BA season logic and provide the dates. (I bring this up because I don't really want to encourage its use.)

Reply:
It's used [here](https://github.com/NREL/resstock/blob/develop/measures/ResStockArguments/measure.rb#L524-L531) in ResStockArguments.

Reply:
Related, https://github.com/NREL/resstock/pull/1254, will be getting BA heating/cooling months from ResStockArguments.

Reply:
Does that mean that ResStock would no longer be passing "BuildingAmerica" to this argument?

Reply:
It still does in the PR. I'm just pointing out that the PR demonstrates it's possible to get BA season dates from ResStockArguments.